### ISSUE TITLE
Doesn't work with latest version of Cursor 

### ISSUE THREAD

**Issue by Kevazy** [Teamial/DevTrack#5]
Hi,

DevTrack is unable to be installed on the latest version of Cursor, which is a VSCode fork. Most VSCode extensions work on it, but this is the first time I have seen this error.

![image](https://github.com/user-attachments/assets/db911c2e-fa52-497c-bb8b-b0db32f986ad)

**Comment by Teamial** [comment]
I just released an update for DevTrack to be compatible with Cursor so restart the extension and let me know if it works

**Comment by Kevazy** [comment]
Thanks for the quick response! I am unable to restart it since it did not let me install it in the first place. I restarted Cursor, but it's the same thing when clicking the `Install` button.

However, I did notice that the extension version appears as `v1.0.2` in the marketplace, while you recently updated the version on here to be `3.0.3`. Are you sure the latest version is uploaded to the marketplace? "Last released" says `2024-12-12, 05:03:41` too

**Comment by Teamial** [comment]
Huh... Yeah the latest version should be uploaded to the market place, at the market place it says the Last updated is 12/14/2024, 4:38:02 PM for me. Maybe try refreshing the page, heres the link: https://marketplace.visualstudio.com/items?itemName=TeannaCole.devtrack

**Comment by Kevazy** [comment]
Yeah, interesting... I was doing it from the extensions menu inside Cursor - do they pre-cache all extensions or something? I was unable to update the page in any way from Cursor, but through the link above I clicked `Download Extension` on the right and dragged the file to the Cursor extension menu and got it installed immediately. So it seems this problem is solved in that case. 

Another thing I noticed now is: when tracking for the first time without having an existing `code-tracking` repository, it seems to commit every single file in the current project folder to the new repository while creating it
